chains/basesepolia/addresses.yaml

```diff
@@ -0,0 +1,11 @@
+staticMerkleRootMultisigIsmFactory: "0xEED8918B9Fb2AEcf2df79f1E07aA77EaC1017dAE"
```

Comment:
Please order these alphabetically to make the linter happy. I believe running `yarn lint:fix` would handle this for you.

Reply:
updated